franklin/users/models.py

```diff
@@ -1,15 +1,78 @@
+import logging
+from django.conf import settings
+from django.dispatch import receiver
 from django.db import models
+from django.db.models.signals import post_save
+from django.utils.translation import ugettext as _
 
+from builder.helpers import make_rest_get_call 
 
-class User(models.Model):
-    """ Represents an authenticated 'user' that allows us to utilize Github
-    webhooks from their repos.
+github_base = 'https://api.github.com/'
+logger = logging.getLogger(__name__)
 
-    :param username: The user's unique username (taken from github)
-    :param github_token: The user's stored github auth token
+
+class UserDetails(models.Model):
+    """ Extra details and functions attached to the default user created with
+    github social signin
+
+    :param user: FK to a unique user
     """
-    username = models.CharField(max_length=30)
-    github_token = models.CharField(max_length=30)
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, related_name='details')
+
```

Comment:
Deleted old user model (not used). Replaced with a UserDetails model that simply links to the vanilla user model for all users. Right now this is just for the `get_user_repos` method. Other user specific details will go here. Did this to avoid extending and/or overriding the vanilla user model like we had done on other project. I think this way is better.